numaprom/__init__.py

```diff
@@ -42,11 +42,25 @@ def __get_logger() -> logger:
     if os.getenv("DEBUG", False):
         filter_levels["numalogic"] = "DEBUG"
         filter_levels["numaprom"] = "DEBUG"
-        logger.add(sink=sink, level="DEBUG", colorize=True, filter=filter_levels, format=formatter)
+        logger.add(
+            sink=sink,
+            level="DEBUG",
+            colorize=True,
+            filter=filter_levels,
+            format=formatter,
+            diagnose=False,
```

Comment:
What does diagnose do?

Reply:
Logger automatically provides diagnose solution. (Logs are becoming extensive too.)

Production should have it as disabled as per the docs.?
1. https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.add